distribution/docs/src/main/resources/content/_running/available-console-commands.adoc

```diff
@@ -0,0 +1,44 @@
+:title: Available System Console Commands
```

Comment:
There a lot more namespaces/commands that aren't included in the documentation. This should be clarified in this section and maybe the name of this section could be updated to something like, DDF Console Commands.

Reply:
added a link to karaf docs for more commands/namespaces